.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+python:
+    - "2.7"
+before_install:
+    - "export DISPLAY=:99"
+    - "sh -e /etc/init.d/xvfb start"
+install:
+    - "pip install -e git://github.com/edx/xblock-sdk.git#egg=xblock-sdk"
+    - "pip install -r requirements.txt"
+    - "pip install -r test-requirements.txt"
+    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements.txt"
+    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/test-requirements.txt"
```

Comment:
@e-kolpakov Try uninstalling the XBlock first (or not installing it at all in requirements.txt):

```
pip uninstall -y xblock-group-project-v2 && python setup.py sdist && pip install dist/xblock-group-project-v2-0.2.tar.gz
```